<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MacBook-Pro-6983.local" ignored="0" name="Tests" tests="7" failures="7" timestamp="21 Dec 2020 23:55:58 GMT" time="91.406" errors="0">
  <testcase name="Validate" time="50.026" classname="com.sample.test.demo.tests.DemoTest">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="-1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: -1
at com.sample.page.demo.PizzaOrderFormPage.pizzaOrder(PizzaOrderFormPage.java:40)
at com.sample.test.demo.tests.DemoTest.Validate(DemoTest.java:45)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Validate -->
  <testcase name="Validate" time="5.554" classname="com.sample.test.demo.tests.DemoTest">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="-1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: -1
at com.sample.page.demo.PizzaOrderFormPage.pizzaOrder(PizzaOrderFormPage.java:40)
at com.sample.test.demo.tests.DemoTest.Validate(DemoTest.java:45)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Validate -->
  <testcase name="Validate" time="4.628" classname="com.sample.test.demo.tests.DemoTest">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="-1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: -1
at com.sample.page.demo.PizzaOrderFormPage.pizzaOrder(PizzaOrderFormPage.java:40)
at com.sample.test.demo.tests.DemoTest.Validate(DemoTest.java:45)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Validate -->
  <testcase name="Validate" time="4.304" classname="com.sample.test.demo.tests.DemoTest">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="-1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: -1
at com.sample.page.demo.PizzaOrderFormPage.pizzaOrder(PizzaOrderFormPage.java:40)
at com.sample.test.demo.tests.DemoTest.Validate(DemoTest.java:45)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Validate -->
  <testcase name="Validate" time="4.675" classname="com.sample.test.demo.tests.DemoTest">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="-1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: -1
at com.sample.page.demo.PizzaOrderFormPage.pizzaOrder(PizzaOrderFormPage.java:40)
at com.sample.test.demo.tests.DemoTest.Validate(DemoTest.java:45)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Validate -->
  <testcase name="Validate" time="7.005" classname="com.sample.test.demo.tests.DemoTest">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="-1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: -1
at com.sample.page.demo.PizzaOrderFormPage.pizzaOrder(PizzaOrderFormPage.java:40)
at com.sample.test.demo.tests.DemoTest.Validate(DemoTest.java:45)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Validate -->
  <testcase name="Validate" time="4.573" classname="com.sample.test.demo.tests.DemoTest">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="-1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: -1
at com.sample.page.demo.PizzaOrderFormPage.pizzaOrder(PizzaOrderFormPage.java:40)
at com.sample.test.demo.tests.DemoTest.Validate(DemoTest.java:45)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Validate -->
</testsuite> <!-- Tests -->
